.github/workflows/lint.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       - name: apt-get dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install libbz2-dev liblzma-dev libreadline-dev libsqlite3-dev
+          sudo apt-get install libbz2-dev liblzma-dev libreadline-dev libsqlite3-dev pdftohtml
```

Comment:
Can you also add this to the wiki: https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Manual-Development-environment-setup

And to the setup.sh script: https://github.com/CAVaccineInventory/vaccine-feed-ingest/blob/main/setup.sh

Reply:
Added to `setup.sh`. If you're willing to temporarily give me write access to the wiki, I'll add it there too.

Reply:
I don't have permission to adjust permissions, but I will add it to the wiki. Thanks!